packages/core/src/browser/tree/tree-widget.tsx

```diff
@@ -178,6 +191,8 @@ export class TreeWidget extends ReactWidget implements StatefulWidget {
         this.node.tabIndex = 0;
     }
 
+    protected parentOfActiveIndentGuideline = new Set<String>();
```

Comment:
can we get rid of it and compute in `renderIndent`?

Reply:
@akosyakov Since we need to clear it and also call it (as it is part of the `addActiveNodeParent`) in both `onSelectionChanged` and `onExpansionChanged`, I couldn't figure out a way to compute it in `renderIndent`. How would you suggest I change it?

Reply:
What is the issue is to get the focused node and check what the rendered node belongs to it? It is much simpler when to read how this field is managed and wonder whether it is correct.